.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
 # https://help.github.com/articles/about-codeowners/
 
 # Default owners
-* @davezuko @kuzhelov @layershifter @levithomason @miroslavstastny @mnajdova
+* @davezuko @kuzhelov @layershifter @levithomason @miroslavstastny @mnajdova @kenotron @jdhuntington @ecraig12345 @dzearing @jasongore
```

Comment:
We can remove this file entirely.  It is outdated/incorrect and we don't have areas of ownership to tie to globs.